app/models/ability.rb

```diff
@@ -0,0 +1,34 @@
+class Ability
+  include CanCan::Ability
+
+  def initialize(user)
+    user ||= User.new
+
+    # Define abilities for the user here. For example:
+    #
+    if user.is? :admin
+      can :manage, :all
+    else
+      can :read, :all
+      can :destroy, :all, author_id: user.id
```

Comment:
- [ ] ` Owner of the post should be able to perform all view, update or destroy own post if Post.author == user` , not only destroy. So please update your code to:
`can :manage, Post, author: user`
`can :manage, Comment, author: user`

Reply:
Done and tested.